### PR TITLE
Feature/34 ng rx add effects

### PR DESCRIPTION
**Pull request by cmendezlopez25**

### CODE REVIEW

File: RideShare/src/styles.css [hunk 1]
```diff
@@ -1,7 +1,34 @@
...
+}
+/* Bold Italic font */
+@font-face {
+  font-family: Futura-PT-Bold-Oblique;
+  src: url('./assets/Futura-PT-Complete/ParaType-Futura-PT-Bold-Oblique.ttf')
+    format('truetype');
+}
 /* Sets all font on the page to match revature style */
-* {
+/* * {
```
Comment (jurczakn) [inline RideShare/src/styles.css]: remove commented out font

File: RideShare/src/styles.css [hunk 2]
```diff
@@ -52,8 +79,8 @@ body {
   font-size: 15px;
   font-weight: 700;
   width: 40%;
-  position: relative;
-  left: 10%;
+  /* position: relative;
```
Comment (jurczakn) [inline RideShare/src/styles.css]: remove

File: RideShare/src/app/models/states/location-state.model.ts [hunk 3]
```diff
@@ -10,12 +10,14 @@ import { HouseLocation } from '../houselocation.model';
  */
 export interface ILocationState {
   /**
-   * allTrainingLocations: Array<TrainingLocation> - contains all of the training locations known in the database
+   * allTrainingLocations: Array<TrainingLocation>
```
Comment (jurczakn) [inline RideShare/src/app/models/states/location-state.model.ts:13]: remove commented out value

File: RideShare/src/app/models/states/location-state.model.ts [hunk 4]
```diff
@@ -10,12 +10,14 @@ import { HouseLocation } from '../houselocation.model';
...
   /**
-   * allTrainingLocations: Array<TrainingLocation> - contains all of the training locations known in the database
+   * allTrainingLocations: Array<TrainingLocation> 
+   * - contains all of the training locations known in the database
    */
   readonly allTrainingLocations: Array<TrainingLocation>;
 
   /**
-   * allHousingLocations: Array<HouseLocation> - contains all of the housing locations known in the database
+   * allHousingLocations: Array<HouseLocation>
```
Comment (jurczakn) [inline RideShare/src/app/models/states/location-state.model.ts:19]: remove

**Review by jurczakn — CHANGES_REQUESTED** [review]

File: RideShare/src/styles.css (hunk 1 above)
Comment (cmendezlopez25) [reply RideShare/src/styles.css]: Removed

File: RideShare/src/styles.css (hunk 2 above)
Comment (cmendezlopez25) [reply RideShare/src/styles.css]: Removed

File: RideShare/src/app/models/states/location-state.model.ts (hunk 3 above)
Comment (cmendezlopez25) [reply RideShare/src/app/models/states/location-state.model.ts:13]: Part of documentation comments, shouldn't remove.

File: RideShare/src/app/models/states/location-state.model.ts (hunk 4 above)
Comment (cmendezlopez25) [reply RideShare/src/app/models/states/location-state.model.ts:19]: Part of documentation comments, shouldn't remove.

**Review by jurczakn — APPROVED** [review]
ok

**Pull request merged**